packages/yoroi-extension/features/support/webdriver.js

```diff
@@ -171,20 +175,23 @@ function CustomWorld(cmdInput: WorldInput) {
     this.getElementBy(locator).getAttribute('value');
 
   this.waitForElementLocated = async (locator: LocatorObject) => {
+    this.webDriverLogger.info(`Webdriver: Waiting for element "${locator.locator}" to be located`);
```

Comment:
I would suggest to replace `${locator.locator}` with `${JSON.stringify(locator)}`